### PR TITLE
Peoplepicker Module

### PR DESCRIPTION
**Pull request by aishwarya56**
Create PeoplePicker Module

### CODE REVIEW

File: FluentUI/build.gradle [hunk 1]
```diff
@@ -35,7 +35,8 @@ dependencies {
     implementation project(':fluentui_listitem')
     implementation project(':fluentui_menus')
     implementation project(':fluentui_others')
-    implementation project(':fluentui_persona')
+    implementation project(':fluentui_others')
```
Comment (aishwarya56) [inline FluentUI/build.gradle:38]: To be fixed

File: FluentUI.Demo/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -109,7 +109,7 @@
 
     <!-- Example theme with Fluent Others theme, you will not have to only look for what rest of the
          modules have set. Majorly everything is set for you in Others theme-->
-    <style name="Base.AppTheme.FluentOthers" parent="Theme.FluentUI">
+    <style name="Base.AppTheme.FluentOthers" parent="Theme.FluentUI.Others">
```
Comment (mnkshukl) [inline FluentUI.Demo/src/main/res/values/styles.xml:112]: why this change , all modules done ?

**Review by mnkshukl — APPROVED** [review]
